dbt_sugar/core/documentation/documentation_helper.py

```diff
@@ -0,0 +1,159 @@
+"""Contains documentation related utils."""
+import logging
+import os
+from pathlib import Path
+from typing import Any, Dict, List
+
+from dbt_sugar.core.yaml_module.yaml_helpers import open_yaml, save_yaml
+
+COLUMN_NOT_DOCUMENTED = "No description for this column."
+MODEL_NOT_DOCUMENTED = "No description for this model."
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Documentation:
+    """Class to administrate the documentation in DBT."""
```

Comment:
minor comment but dbt is not supposed to be capitalised as per their brand guidelines.